prisma/schema.prisma

```diff
@@ -0,0 +1,43 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Buyer {
+  bid                     String         @id @default(cuid())
+  buyer_name              String?
+  total_transaction_value Int?
+  email                   String?
+  saved_meathod           String
+  fav_meathod             String
+  createdAt               DateTime       @default(now())
+  updatedAt               DateTime       @updatedAt
+  Transactions            Transactions[]
+}
+
+model Transactions {
+  tid       String      @id @default(cuid())
+  buyerID   Int?
+  orderOn   DateTime
+  cartValue Int?
+  customer  Buyer       @relation(fields: [buyerId], references: [bid])
+  buyerId   String
+  createdAt DateTime    @default(now())
+  updatedAt DateTime    @updatedAt
+  Discounts Discounts[]
```

Comment:
1 Transaction should ideally allow 1 discount! I don't think its a great idea to keep many to many here!